statistics/cmsketch_test.go

```diff
@@ -127,3 +154,60 @@ func (s *testStatisticsSuite) TestCMSketchCoding(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(lSketch.Equal(rSketch), IsTrue)
 }
+
+func (s *testStatisticsSuite) TestCMSketchTopN(c *C) {
+	tests := []struct {
+		zipfFactor float64
+		avgError   uint64
+	}{
+		// If no significant most items, TopN may will produce results worse than normal algorithm.
+		{
+			zipfFactor: 1.0000001,
+			avgError:   48,
```

Comment:
Can we make the error of low frequency and high frequency separated?

Reply:
Difficult, when zipfFactor is 1.0000001 and 1.1, avgerror are both around 30. I wonder if this is the lower bound of avgerror, and will do some tests.

Reply:
OK.